includes/api/class-wc-rest-products-controller.php

```diff
@@ -1078,6 +1078,11 @@ protected function set_product_images( $product, $images ) {
 				if ( ! empty( $image['name'] ) ) {
 					wp_update_post( array( 'ID' => $attachment_id, 'post_title' => $image['name'] ) );
 				}
+
+				// Set for future reference
```

Comment:
Missing a period here.

Reply:
Do you say about the comment?
`Set the image source if present, for future reference.`.

Reply:
Yes, just need to end with a period.
But this new text works great too.

Reply:
Ok, this correction has be resolved.